transmogrifier/sources/transformer.py

```diff
@@ -118,10 +118,12 @@ def get_required_fields(self, xml: Tag) -> dict:
         """
         source_record_id = self.get_source_record_id(xml)
         title = self.get_valid_title(source_record_id, xml)
+        source_link = self.get_source_link(self.source_base_url, source_record_id, xml)
+        timdex_record_id = self.get_timdex_record_id(self.source, source_record_id, xml)
```

Comment:
Optional: you could call these directly in the dict

Reply:
Happy to change if you'd like, and I considered that too.  But at the risk of a being a bit more verbose, I thought it helped break it up a bit.  There is some important stuff happening here -- establishing record title, primary timdex id, link, etc. -- and I found it helpful to see those steps as distinct statements vs embedded in a dictionary.

Reply:
Optional is always just a suggestion 🙂